index.js

```diff
@@ -117,22 +165,44 @@ var Umzug = module.exports = redefine.Class({
 
         return all.filter(function (migration) {
           return executedFiles.indexOf(migration.file) === -1;
-        }).sort(function (a, b) {
```

Comment:
It should not be a bug fix since it has introduced and last changed in #1. But it could be for Windows. As it makes no harm, I'll move it to `_findMigrations`.